rules/enforce-svg-import.js

```diff
@@ -0,0 +1,41 @@
+const reportError = (context, node) => {
+  const parts = node.source.raw.split(".")
+  const name = parts[1]
+  .replace(/[A-Z\xC0-\xD6\xD8-\xDE]?[a-z\xDF-\xF6\xF8-\xFF]+|[A-Z\xC0-\xD6\xD8-\xDE]+(?![a-z\xDF-\xF6\xF8-\xFF])|\d+/g, (word) => word.charAt(0).toUpperCase() + word.slice(1))
```

Comment:
Could you add a comment above what it does? I really have to look up what `\xC0` for example is and I don't have a clue why it is this specific character that is in an capturing group, so cannot validate whether we miss something or whether there are edge-cases.

Reply:
Now that I think about it, if you ever wanted a valid case for writing a unit-test, the above is one!